From e32fe30d0cf7c1f7045ac0bd29322e7fb4feb5c8 Mon Sep 17 00:00:00 2001
From: Daniel Stenberg <daniel@haxx.se>
Date: Tue, 23 Mar 2010 23:25:04 +0100
Subject: [PATCH] enabled valgrind

I ran it now successfully and it helped to pinpoint a libssh2
memory leak!
---
 tests/data/test623 |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/tests/data/test623 b/tests/data/test623
index 38a41d2..19e505b 100644
--- a/tests/data/test623
+++ b/tests/data/test623
@@ -36,8 +36,6 @@ for ssh upload test
 <errorcode>
 79
 </errorcode>
-<valgrind>
-disable
-</valgrind>
+
 </verify>
 </testcase>
-- 
1.6.6.1

